### PR TITLE
[libosmium] update to 2.19.0

### PR DESCRIPTION
**Pull request by IldarKhayrutdinov**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by IldarKhayrutdinov** [comment]
@microsoft-github-policy-service agree

**Comment by IldarKhayrutdinov** [comment]
oh, I missed, fixed, but failures `microsoft.vcpkg.pr `, `microsoft.vcpkg.pr (x86_windows)`

File: ports/libosmium/portfile.cmake [hunk 1]
```diff
@@ -2,8 +2,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO osmcode/libosmium
-    REF v2.18.0
-    SHA512 bfe4d5b08e284abb21280205437c62d260948c8585cc8d10f0c3496fd29df1ff6c6bc436283207f292d6a8af911835ebef64403c3fde2a262b55fde5296c13ce
+    REF v2.19.0
```
Comment (jimwang118) [inline ports/libosmium/portfile.cmake:5]:
```suggestion
    REF "v${VERSION}"

```

**Review by jimwang118 — CHANGES_REQUESTED** [review]

**Comment by jimwang118** [comment]
> oh, I missed, fixed, but failures `microsoft.vcpkg.pr `, `microsoft.vcpkg.pr (x86_windows)`

You rerun the command `./vcpkg x-add-version {portname} --overwrite-version` and resubmit.

**Review by jimwang118 — APPROVED** [review]

**Pull request merged**